rust/syntect-plugin/examples/make_manifest.rs

```diff
@@ -41,7 +43,28 @@ fn parse_name_and_version() -> Result<(String, String), io::Error> {
 }
 
 fn main() -> Result<(), io::Error> {
-    let syntax_set = SyntaxSet::load_defaults_newlines();
+    let config_values = ConfigManager::new(None, None);
+    let syntax_set = match &config_values.get_syntaxes_path() {
```

Comment:
ah, I see what you're doing.

I don't think this file should load the config; it should be self contained.

This works fine for allowing us to create a custom default syntax set, but not for loading syntaxes dynamically.

I think we should treat these features independently; doing the custom defaults is a good first step, and it will give you a better understanding for how we should go about loading them dynamically (we might need to rethink how manifests work, for that).